source/extensions/filters/http/health_check/health_check.cc

```diff
@@ -119,52 +119,53 @@ void HealthCheckFilter::onComplete() {
   Http::Code final_status = Http::Code::OK;
   const std::string* details = &RcDetails::get().HealthCheckOk;
   bool degraded = false;
+  stats_->request_total_.inc();
   if (context_.healthCheckFailed()) {
     callbacks_->streamInfo().setResponseFlag(StreamInfo::CoreResponseFlag::FailedLocalHealthCheck);
     final_status = Http::Code::ServiceUnavailable;
     details = &RcDetails::get().HealthCheckFailed;
+    stats_->failed_.inc();
   } else {
     if (cache_manager_) {
       const auto status_and_degraded = cache_manager_->getCachedResponse();
       final_status = status_and_degraded.first;
       details = &RcDetails::get().HealthCheckCached;
       degraded = status_and_degraded.second;
+      stats_->cached_response_.inc();
     } else if (cluster_min_healthy_percentages_ != nullptr &&
                !cluster_min_healthy_percentages_->empty()) {
-      // Check the status of the specified upstream cluster(s) to determine the right response.
```

Comment:
Why were the comments here and below removed?

Reply:
I have added them back, Thanks!